requirements.txt

```diff
@@ -7,7 +7,7 @@ mako
 chardet
 six
 setproctitle
-pywin32; sys_platform == 'win32'
+pypiwin32; sys_platform == 'win32'
```

Comment:
Please provide details in this commit message as to what if being fixed by this change. From my perspective both provide pywin32 and I havn't seen any problems.

Reply:
In further testing this appears to be a issue with modules getting installed with the setup.py script(at least during a "python setup.py develop") After this setup pywin32 is "installed" but doesn't work, when going through the uninstall process you can see theres a big difference in files. and since it is "installed" following up with a pip install -r requirements.txt just says its already installed. However if you uninstall pywin32 then install with pip -r... it installs properly.